pkg/engine/validate/utils.go

```diff
@@ -0,0 +1,55 @@
+package validate
+
+import (
+	"container/list"
+	"github.com/nirmata/kyverno/pkg/engine/anchor"
+)
+
+// Checks if pattern has anchors
+func hasNestedAnchors(pattern interface{}) bool {
```

Comment:
This method is also used in overlay, if it's common utility function, let's extract it to pkg/engine/common, then refer to the same method from everywhere else.

Reply:
its different than that